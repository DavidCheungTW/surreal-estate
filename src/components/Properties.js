import React from "react";

const Properties = () => (
  <div>
    <h3>Properties Page</h3>
  </div>
);

export default Properties;
