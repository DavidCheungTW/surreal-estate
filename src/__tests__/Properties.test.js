import React from "react";
import { fireEvent, render, screen, waitFor } from "@testing-library/react";
import renderer from "react-test-renderer";
import Properties from "../components/Properties";

describe("Properties", () => {
  test("Renders as expected", () => {
    const rendered = renderer.create(<Properties />);

    expect(rendered).toMatchSnapshot();
  });

  test("Assert Properties Page Heading", () => {
    render(<Properties />);

    expect(screen.getByText(/Properties Page/i)).toBeTruthy();
    expect(screen.getByText(/Properties Page/i)).toBeInstanceOf(
      HTMLHeadingElement
    );
  });
});
