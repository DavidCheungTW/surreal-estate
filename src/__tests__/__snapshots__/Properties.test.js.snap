// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Properties Renders as expected 1`] = `
<div>
  <h3>
    Properties Page
  </h3>
</div>
`;
